docs/resources/cluster.md

```diff
@@ -182,11 +180,11 @@ init_scripts {
 }
 ```
 
-Attributes are the same as for `cluster_log_conf` configuration block.
+Attributes are the same as for the `cluster_log_conf` configuration block.
 
 ## aws_attributes
 
-`aws_attributes` optional configuration block contains attributes related to [clusters running on Amazon Web Services](https://docs.databricks.com/clusters/configure.html#aws-configurations). If not specified at cluster creation, a set of default values will be used. It is advised to keep all common configurations in [Cluster Policies](cluster_policy.md) to maintain control of the environments launched.
+`aws_attributes` optional configuration block contains attributes related to [clusters running on Amazon Web Services](https://docs.databricks.com/clusters/configure.html#aws-configurations). If not specified at cluster creation, a set of default values will be used. 
```

Comment:
Can we specify the partial list of the attributes? Is it all or nothing?  Also, documentation doesn't talk about specific configuration parameters, so it may not be clear what could be specified here.

Reply:
it does cover specific params below